cloud_chatops/src/pr_reminder.py

```diff
@@ -0,0 +1,205 @@
+"""This module handles the posting of messages to Slack using the Slack SDK WebClient class."""
+
+from typing import List, Dict
+from datetime import datetime, timedelta
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+from src.read_data import get_token, get_user_map, get_repos
+from src.get_github_prs import GetGitHubPRs
+
+
+class PostPRsToSlack:
+    """
+    This class handles the Slack posting.
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.repos = get_repos()
+        self.client = WebClient(token=get_token("SLACK_BOT_TOKEN"))
+        self.slack_ids = get_user_map()
+        self.prs = GetGitHubPRs(get_repos(), "stfc").run()
```

Comment:
To make this easier - we could potentially have`GetGitHubPRs.run()` return a list of dataclasses - where each dataclass contains information about a PR